modules/distribution/resources/migration-1.8.0_to_1.9.0/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/client/MigrateFrom18to19.java

```diff
@@ -0,0 +1,702 @@
+                            apiProviderName);
+
+                    if (registry.resourceExists(swagger12location)) {
+                        registry.delete(APIConstants.API_DOC_LOCATION);
+                        log.info("Resource deleted from the registry.");
+
+                    }
+
+                }
+            }
+        } catch (APIManagementException e) {
+            ResourceUtil.handleException("API Management Exception occurred while migrating rxt.", e);
+        } catch (UserStoreException e) {
+            ResourceUtil.handleException("Error occurred while reading tenant admin.", e);
+        } catch (RegistryException e) {
+            ResourceUtil.handleException("Error occurred while accessing the registry.", e);
+        }
+        if (log.isDebugEnabled()) {
+            log.debug("old resources cleaned up.");
+        }
+    }
+
+    /**
+     * This method is used to migrate sequence files
+     * This adds cors_request_handler to sequences
+     *
+     * @throws APIMigrationException
+     */
+    void sequenceMigration() throws APIMigrationException {
+        String repository = CarbonUtils.getCarbonRepository();
+        String TenantRepo = CarbonUtils.getCarbonTenantsDirPath();
+        for (Tenant tenant : tenantsArray) {
+            String SequenceFilePath;
+            if (tenant.getId() != MultitenantConstants.SUPER_TENANT_ID) {
+                SequenceFilePath = TenantRepo + "/" + tenant.getId() +
+                        "/synapse-configs/default/sequences/";
+            } else {
+                SequenceFilePath = repository + "synapse-configs/default/sequences/";
+            }
+            try {
+                FileUtils.copyInputStreamToFile(MigrateFrom18to19.class.getResourceAsStream(
+                                "/18to19Migration/sequence-scripts/_cors_request_handler.xml"),
+                        new File(SequenceFilePath + "_cors_request_handler.xml"));
+                ResourceUtil.copyNewSequenceToExistingSequences(SequenceFilePath, "_auth_failure_handler_");
+                ResourceUtil.copyNewSequenceToExistingSequences(SequenceFilePath, "_throttle_out_handler_");
+                ResourceUtil.copyNewSequenceToExistingSequences(SequenceFilePath, "_token_fault_");
+                ResourceUtil.copyNewSequenceToExistingSequences(SequenceFilePath, "fault");
+            } catch (IOException e) {
+                ResourceUtil.handleException("Error occurred while reading file to copy.", e);
+            }
+        }
+    }
+
+
+    /**
+     * This method is used to migrate synapse files
+     * This changes the synapse api and add the new handlers
+     *
+     * @throws APIMigrationException
+     */
+
+    void synapseAPIMigration() throws APIMigrationException {
+        String repository = CarbonUtils.getCarbonRepository();
+        String tenantRepository = CarbonUtils.getCarbonTenantsDirPath();
+        for (Tenant tenant : tenantsArray) {
+
+            String SequenceFilePath;
```

Comment:
Variable namin is incorrect. Variable name should start from a lowercase letter.